features/kafka/producer/src/main/java/org/opennms/features/kafka/producer/collection/KafkaPersister.java

```diff
@@ -54,20 +56,37 @@ public class KafkaPersister implements Persister {
 
     private CollectionSetMapper collectionSetMapper;
 
+    private final ServiceParameters m_params;
+
     private KafkaProducer<String, byte[]> producer;
     
     private String topicName = "metrics";
 
+    public KafkaPersister(ServiceParameters params) {
+        m_params = params;
+    }
+
+    /**
+     * Only for Mockito, DO NOT USE OTHERWISE.
+     */
+    public KafkaPersister() {
```

Comment:
Is this getting used somewhere ?  Why can't it be handled with other constructor ?